src/api/gen/api.ts

```diff
@@ -634,7 +689,7 @@ export const DefaultApiFp = function(configuration?: Configuration) {
          */
         async getnativev1me(basePath: string, options?: any): Promise<UserProfileResponse> {
             const localVarFetchArgs = await DefaultApiFetchParamCreator(configuration).getnativev1me(options);
-            const response = await safeFetch(basePath + localVarFetchArgs.url, localVarFetchArgs.options)
+            const response = await safeFetch(basePath + localVarFetchArgs.url, localVarFetchArgs.options, api)
```

Comment:
better than **this** !

on se protège des bizarreries de JS en passant le bon objet une fois pour toutes